lib/elastic_record/index.rb

```diff
@@ -84,14 +79,14 @@ def configure(&block)
     end
 
     def get(end_path, json = nil)
-      connection.json_get "/#{alias_name}/#{type}/#{end_path}", json
+      doctype = doctypes.first
```

Comment:
This is the only problem with doctypes right now:  sometimes, operations have to specify a doctype, so we need to know which one.